### PR TITLE
Fix: Rename blocks for CareerSocius General

### PR DESCRIPTION
**Pull request by kvihashini**
# Notion Task

https://www.notion.so/70204328dfbc4abfa76a39d9c3b50169?v=2db6790b04df468cb2f1e85a6c117567&p=703e464b72114ed4b9286c2d22597e56 

# Changes Made In This PR

- Renamed blocks from 'Block 1' and 'Block 2' to 'Content' and 'Additional Information' if CareerSocius General template is chosen

# Screenshots

**Before:**
Screenshot 2022-04-10 at 7 05 04 PM

**After:**
Screenshot 2022-04-10 at 6 55 57 PM

### CODE REVIEW

**Review by millasml — CHANGES_REQUESTED** [review]
getting an error when i try to choose the general template in the preview link:

image

File: src/components/builder/left/sections/Layout.js [hunk 1]
```diff
@@ -121,6 +121,17 @@ const Layout = () => {
...
   };
 
+  const wallStreetLayout = (ind) => {
+    switch (ind) {
+      case '0':
+        return `${t('builder.templates.content')}`;
+      case '1':
+        return `${t('builder.templates.additionalInfo')}`;
+      default:
+        throw new Error();
```
Comment (millasml) [inline src/components/builder/left/sections/Layout.js:131]: change this line to
```
`${t('builder.layout.block')} ${ind + 1}`
```

File: src/components/builder/left/sections/Layout.js [hunk 2]
```diff
@@ -139,7 +150,9 @@ const Layout = () => {
                 >
                   <div className="grid gap-3">
                     <span className="uppercase font-semibold text-xs">
-                      {t('builder.layout.block')} {ind + 1}
+                      {template === 'wallStreet'
```
Comment (millasml) [inline src/components/builder/left/sections/Layout.js:153]: can you use the enum from `TemplateEnum` in `Template.ts`?

File: src/components/builder/left/sections/Layout.js [hunk 3]
```diff
@@ -121,6 +121,17 @@ const Layout = () => {
     setResetLayoutText(t('builder.layout.reset'));
   };
 
+  const wallStreetLayout = (ind) => {
```
Comment (millasml) [inline src/components/builder/left/sections/Layout.js:124]: lets rename this to `getWallStreetBlockName`

**Review by millasml — CHANGES_REQUESTED** [review]
hmm it doesn't seem to work :( 

image

maybe its because your switch case are strings, when ind is probably a number?

**Comment by kvihashini** [comment]
> hmm it doesn't seem to work :(
> 
> image
> 
> maybe its because your switch case are strings, when ind is probably a number?

ah right, that makes sense! will change it, thank you!

**Review by millasml — APPROVED** [review]

**Pull request merged**